app/src/main/java/org/stepic/droid/ui/adapters/view_hoders/CourseItemViewHolder.kt

```diff
@@ -191,14 +196,11 @@ class CourseItemViewHolder(
                 .fitCenter()
                 .into(imageViewTarget)
 
-        if (course.learnersCount > 0) {
+        val needShowLearners = course.learnersCount > 0
+        if (needShowLearners) {
             learnersCountText.text = String.format(Locale.getDefault(), "%d", course.learnersCount)
-            learnersCountContainer.visibility = View.VISIBLE
-        } else {
-            learnersCountContainer.visibility = View.GONE
         }
```

Comment:
If the course doesn't have learners person icon will be shown along with some number of learners from a different course due to reusing of views in RecyclerView.
<img src="https://user-images.githubusercontent.com/3856523/32649291-a0a5a21c-c60a-11e7-8001-8efb3345d06a.png" width="300">
<img src="https://user-images.githubusercontent.com/3856523/32649294-a2f84f42-c60a-11e7-88d3-a611f3734fbb.png" width="300">

Reply:
Nice, thanks